railties/lib/rails/application/finisher.rb

```diff
@@ -173,6 +173,22 @@ def self.complete(_state)
         end
       end
 
+      initializer :add_builtin_route do |app|
+        if Rails.env.development?
+          app.routes.prepend do
+            get "/rails/info/properties" => "rails/info#properties", internal: true
+            get "/rails/info/routes"     => "rails/info#routes", internal: true
+            get "/rails/info"            => "rails/info#index", internal: true
```

Comment:
```suggestion
            get "/rails/info/properties" => "rails/info#properties", internal: true
            get "/rails/info/routes"     => "rails/info#routes",     internal: true
            get "/rails/info"            => "rails/info#index",      internal: true
```

Let's line this internal arguments up while we're here.